src/createMollieClient.ts

```diff
@@ -58,6 +59,25 @@ import SettlementsBinder from './binders/settlements/SettlementsBinder';
 import SubscriptionsBinder from './binders/subscriptions/SubscriptionsBinder';
 import SubscriptionPaymentsBinder from './binders/subscriptions/payments/SubscriptionPaymentsBinder';
 
+/**
+ * Returns an error message (string) similar to `"Parameter "×" is null."` if a property with the passed key exists in
+ * the passed object, or `null` if said property does not exist.
+ */
+const describeFalsyOption = run(
+  new Map([
+    [undefined, 'undefined'],
+    [null, 'null'],
+    ['', 'an empty string'],
+  ]) as Map<any, string>,
+  descriptions =>
+    function describeFalsyOption<O extends Record<string, unknown>>(object: O, key: keyof O & string) {
+      if (key in object == false) {
+        return null;
+      }
+      return `Parameter "${key}" is ${descriptions.get(object[key]) ?? object[key]}.`;
+    },
+);
+
```

Comment:
It might be a good idea to extract the credentials check to a separate file. What do you think?

Reply:
Yes, that's sensible.